lib/cocoapods-core/specification/linter/analyzer.rb

```diff
@@ -43,6 +43,7 @@ def check_attributes
           attributes_keys = Pod::Specification::DSL.attributes.keys.map(&:to_s)
           platform_keys = Specification::DSL::PLATFORMS.map(&:to_s)
           valid_keys = attributes_keys + platform_keys
+          valid_keys << 'test_specification'
```

Comment:
thoughts welcome, the analyzer cannot find this key so its tests fail, should I add `test_specification` to `dsl.rb`?